### ISSUE TITLE
touch input not recognized (react-native v0.15.x)

### ISSUE THREAD

**Issue by webwelten** [iodine/react-native-drawer-layout#15]
Since I updated to react-native version v0.15.0 from v0.13.2, I've had the problem that only the swipe gesture (to open the menu) is recognized but NOT the touch input on the child views of the drawer layout. As I explored the code I saw that the overlay is always on top of the child views and consuming all my touch inputs.

Now I wonder if that's a problem in my code, some side effect maybe OR if there were some changes with react native causing this issue. 

```
  <View style={{flex: 1, backgroundColor: 'transparent'}} {...this._panResponder.panHandlers}>

    <Animated.View style={styles.main}>
      {this.props.children}
    </Animated.View>

    <Animated.View style={[styles.overlay, animatedOverlayStyles]} />

    <Animated.View style={[styles.drawer, dynamicDrawerStyles, animatedDrawerStyles]}>
      {this.props.renderNavigationView()}
    </Animated.View>
  </View>
```

**Comment by JonnyBGod** [comment]
Having the same problem.
Everything was woking fine up until RN 0.15

**Comment by dbanck** [comment]
I've had the same problem. b9008691ea614af238296aa278674659a73645c7 fixed it for me.

**Comment by DanielMSchmidt** [comment]
@webwelten @JonnyBGod Does this issue still occurs with the current version or may I close this issue?

**Comment by marcoacierno** [comment]
@DanielMSchmidt Yep, it still happens to me.  (I'm using the repo as dependency since the keyboard fix is not available yet on npm) (react native 0.17)

**Comment by JonnyBGod** [comment]
@DanielMSchmidt yes this problem persists. Have to manually add `pointerEvents="none"` like @dbanck comment every time I update this library.

**Comment by webwelten** [comment]
We also have the problem again although I remember it had worked in the meantime.
(react native 0.17.0)

**Comment by DanielMSchmidt** [comment]
Should be fixed now (0.3.5)
